src/promptflow/promptflow/azure/operations/_run_operations.py

```diff
@@ -73,8 +73,8 @@ class RunOperations(_ScopeDependentOperations):
     attaches it as an attribute.
     """
 
-    DATASTORE_PATH_PATTERN = re.compile(r"azureml://datastores/(?P<datastore>[\w/]+)/paths/(?P<path>.*)$")
-    ASSET_ID_PATTERN = re.compile(r"azureml:/.*?/data/(?P<name>.*?)/versions/(?P<version>.*?)$")
+    _DATASTORE_PATH_PATTERN = re.compile(r"azureml://datastores/(?P<datastore>[\w/]+)/paths/(?P<path>.*)$")
```

Comment:
Is this to avoid those constants appear in doc string? Maybe add some comment for this?

Reply:
Seems those constants are not only for RunOperations, may consider to put them in a common file.

Reply:
Yes it's for hiding constants from doc site.
Currently they are only used in run operations, and since there are regex objects not literal strings, may consider put them in common file when they are acutually used outside of RunOperations.